core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/InstrumentationDirectives.scala

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.dsde.rawls.metrics
+
+import spray.routing.Directive0
+import spray.routing.directives.MiscDirectives.requestInstance
+import spray.routing.directives.BasicDirectives.pass
+
+trait InstrumentationDirectives extends RawlsInstrumented {
+
+  // TODO can I add timing here (Anu's story) - google "scala spray request timing"
+
+  // TODO: "expand nothing" is ugly but the base name will get added regardless so I don't want to do it twice
+  lazy val builder: ExpandedMetricBuilder = ExpandedMetricBuilder.expand("")
+
+  def instrumentRequest: Directive0 = requestInstance flatMap { request =>
+    httpRequestCounter(builder)(request).inc()
```

Comment:
my pref. would be a version of `httpRequestCounter` that doesn't require an existing `ExpandedMetricBuilder `